python/kvikio/kvikio/defaults.py

```diff
@@ -200,3 +200,46 @@ def set_gds_threshold(nbytes: int):
         yield
     finally:
         gds_threshold_reset(old_value)
+
+
+def bounce_buffer_size() -> int:
+    """Get the size of the bounce buffer used to stage data in host memory.
+
+    Set the value using `bounce_buffer_size_reset()` or by setting the
+    `KVIKIO_BOUNCE_BUFFER_SIZE` environment variable. If not set, the
+    value is 16 MiB.
+
+    Return
+    ------
+    nbytes : int
+        The bounce buffer size in bytes.
+    """
+    return kvikio._lib.defaults.bounce_buffer_size()
+
+
+def bounce_buffer_size_reset(nbytes: int) -> None:
+    """Reset the size of the bounce buffer used to stage data in host memory.
+
+    Parameters
+    ----------
+    nbytes : int
+        The bounce buffer size in bytes.
+    """
+    kvikio._lib.defaults.bounce_buffer_size_reset(nbytes)
+
+
+@contextlib.contextmanager
+def set_bounce_buffer_size(nbytes: int):
```

Comment:
What happens if a negative value is used? Does it wrap to a huge value? Do huge values cause errors? Let’s check the behavior and error appropriately.